docs/readmes/orc8r/deploy_build.md

```diff
@@ -107,9 +133,13 @@ The valid deployment type options are
 Run the package script
 
 ```bash
-$ ${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa  # or chosen deployment type
+$ ${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa # or chosen deployment type
 
 ...
 
 Uploaded orc8r charts successfully.
 ```
+You can add `-v` option to overwrite the versions of the chart.
+```bash
+$ ${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa  -v 1.5
```

Comment:
Here and elsewhere: don't include the leading `$` unless you also include the output of the script below it
```suggestion
${MAGMA_ROOT}/orc8r/tools/helm/package.sh -d fwa  -v 1.5
```